src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Icu.cs

```diff
@@ -287,6 +306,12 @@ private unsafe string IcuGetTimeFormatString(bool shortFormat)
         internal static bool IcuIsEnsurePredefinedLocaleName(string name)
         {
             Debug.Assert(!GlobalizationMode.UseNls);
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+            if (GlobalizationMode.Hybrid)
+            {
+                return Interop.Globalization.IsPredefinedLocaleNative(name);
+            }
```

Comment:
We should maintain consistency
```
#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
    if (Hybrid)
    {
        HybridImplementation
    }
    else
    {
        NotHybridImplementation
    }
#else
    NotHybridImplementation
#endif
```

Or we could make all of them (in a separate PR to address all instances and not bloat this PR)
```
#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
    if (Hybrid)
    {
        HybridImplementation
    }
#endif
    NotHybridImplementation
```

Reply:
First code block is used when it is not possible to return in `HybridImplementation`

#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
    if (Hybrid)
    {
        HybridImplementation
    }
    else
    {
        NotHybridImplementation
    }
#else
    NotHybridImplementation
#endif

Second code block is used when we can return in `HybridImplementation` part, otherwise `NotHybridImplementation` code will overwrite values
```
#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
    if (Hybrid)
    {
        HybridImplementation
    }
#endif
    NotHybridImplementation
```
For consistency we can use everywhere first way (second way will not be possible for all cases).

Reply:
Another possibility is:


```
#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
    if (Hybrid)
    {
        HybridImplementation
    }
    else
#endif
    {
        NotHybridImplementation
    }
```

This avoids duplicating the `NotHybridImplementation` logic.